exercises/en/exc_penguins.py

```diff
@@ -0,0 +1,7 @@
+import altair as alt
+import pandas as pd
+
+
+penguins = pd.read_csv('data/penguins.csv')
+print(penguins.iloc[:5, :4])
+print(penguins.iloc[:5, 4:])
```

Comment:
When just outputting the data frame makes the column names messy, let's use `.info` instead of numerical subsetting, as this will show the column names and types, and be more like what we'd actually like the students to do IRL.
```suggestion
penguins.info()
```